src/components/search-results/index.js

```diff
@@ -243,7 +243,7 @@ class SearchResults extends Component {
     const gridStyle = searchComplete && searchItems.length === 0 ? hideGridStyle : showGridStyle;
     return (
       <div className='gridContainer'>
-        {!isInitialTag && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
+        {!isInitialTag && ranking && searchComplete && hotelItems.length === 0 ? noHotelsErrorMessage : ''}
```

Comment:
Actually, It would be more complex, because having no ranking doesn't mean we actually have hotels (so message should not be skipped maybe). 
I will change it a little bit. 
Thanks